streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -221,7 +221,7 @@ public long size() {
      * @return this updated builder
      * @throws IllegalArgumentException if the {@code afterWindowEnd} is negative or can't be represented as {@code long milliseconds}
      * @throws IllegalStateException if {@link #grace(Duration)} is called after {@link #ofTimeDifferenceAndGrace(Duration, Duration)} or {@link #ofTimeDifferenceWithNoGrace(Duration)}
-     * @deprecated since 3.0. Use {@link #ofTimeDifferenceAndGrace(Duration, Duration)} instead
+     * @deprecated Since 3.0. Use {@link #ofTimeDifferenceAndGrace(Duration, Duration)} instead.
```

Comment:
Hi @mjsax 
I notice that this has been deprecated since 3.0 which is passed one year since it was deprecated.
Do we need to delete this one in 4.0?

Reply:
Thanks for paying attention! We decided to keep this old API on purpose and not remove with 4.0.

Cf https://issues.apache.org/jira/browse/KAFKA-13813